src/sessions/ses-friends-s6-fixations.py

```diff
@@ -0,0 +1,20 @@
+from ..tasks import video
+
+TASKS = []
+
+# de-comment line 7 when done piloting
+#for episode in list(range(1, 4)): # to test, just not to download the entire season for local tests
+for episode in list(range(1, 16))+list(range(17,25)):
+    segments = "abcd" if episode in (15,24) else "ab"
+    for segment in segments:
+        TASKS.append(
+            video.SingleVideo(
+                "data/videos/friends/s6/friends_s06e%02d%s.mkv" % (episode, segment),
+                aspect_ratio=4 / 3.0,
+                startend_fixduration=0.0,#2.0,
```

Comment:
No. The start/end fixations are not really needed, but also I wanted to make the loop more efficient : either it enters the _startend_fixduration loop or the _inmovie_fixations loop but not both (I was trying to avoid the glitch)
.

Reply:
did it, thanks!